ChangeLog

```diff
@@ -16,6 +16,11 @@ Release date: TBA
 
   Closes #85, #2615
 
+* Fixed a false positive for ``assigning-non-slot`` when the slotted class
```

Comment:
Could you add an entry to the changelog stored in the `2.13` file?

Reply:
To check: the `docs/whatsnew/2.13.rst` file?

Reply:
Yes! Sorry, should have been clearer about that.

Reply:
No problem, done in b78b0278!